packages/types/lib/runner/sdk.ts

```diff
@@ -30,6 +30,7 @@ export interface NangoProps {
     debug: boolean;
     startedAt: Date;
     endUser: { id: number; endUserId: string | null; orgId: string | null } | null;
+    heartbeatTimeoutSecs?: number | undefined;
```

Comment:
I am not a huge fan of leaking task scheduling info into the nango props and add scheduling related logic into the runner. Especially since we have not confirmed with a real example that this scenario where heartbeat and abort are both not happening

Reply:
Locally, you can have a zombie task; you just need to kill the orchestrator (or make it unavailable). If you combine that with an unreliable network, even without repro, I wouldn't trust two back-to-back HTTP calls to never fail.
You have a better strategy?

Reply:
I don't have a better strategy and I agree that in theory it could happen but I don't think the scenario this PR is fixing is what happen today with our customer.